### ISSUE TITLE
[Request] Update known missing wrappers/fixes

### ISSUE THREAD

**Issue by CodexHere** [nativewrappers/fivem-client#21]
**Is your feature request related to a problem? Please describe.**
I've noticed the former upstream repo from d0p3t has some forks that have a bunch of additions this repo doesn't, and vice versa. However, this repo/npm module looks more official, and updated more frequently.

**Describe the solution you'd like**
Grab the updates from the forks and merge in fixes/updates

**Describe alternatives you've considered**
I could possibly provide a PR once I get time to hand cherry-pick the changes. It might take a few days, but didn't want to start if there's no desire to consume the contributions.

**Additional context**
Let me know if either a) there's plans to gather changes from existing contributors, or b) it'd be acceptible to create a PR with the changes myself.

Examples of changes found in other forks:

* Function documentation
* Updated Enums
* Updated namespaces
* Useful functions (i.e., getting vehicles in a radius)
* Updated missing Entities, and minor fixes to existing ones

**Comment by AvarianKnight** [comment]
> b) it'd be acceptible to create a PR with the changes myself.

This be fine with me as long as they maintain linting and consistency with the project, do note there might be some (or a lot) of breaking changes coming to this library

**Comment by CodexHere** [comment]
Hmm yeah I don't mind working through all the linting and such, but as for the breaking changes it makes me want to wait until that happens. Worst question ever, I know, but do you have an idea of when that might happen?

**Comment by AvarianKnight** [comment]
Probably not soon I have a bunch of other stuff I have to do

**Comment by CodexHere** [comment]
Fair enough. I'm still learning some GTA things, and when I get to a stopping point I'll start working on aggregating changes across forks.

Hopefully it's not a lost cause 😅
